### PR TITLE
Redesign CDN deploy to upload to a tagged version

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: While uploading CDN files, they target a tagged directory instead of an `X.Y.Z` version. In case of a stable (`staging`) release, the rename will be performed later in the release process, allowing us to restart the entire process without affecting users using CDN.

---

### Additional information

* Part of: https://github.com/cksource/ckeditor5-internal/issues/3811.
* Requires: https://github.com/ckeditor/ckeditor5-dev/pull/1019.

### CODE REVIEW

File: scripts/release/publishpackages.mjs [hunk 1]
```diff
@@ -80,19 +80,6 @@ const tasks = new Listr( [
 					'ckeditor5'
 				]
 			} );
-		},
-		retry: 3
-	},
-	{
-		title: 'Checking if packages that returned E409 error code were uploaded correctly.',
-		task: async ( _, task ) => {
-			return releaseTools.verifyPackagesPublishedCorrectly( {
```
Comment (martnpaneq) [inline scripts/release/publishpackages.mjs:89]: What about this check? Are we not going to check for 409 errors any more?

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Review by LukaszGudel — DISMISSED** [review]

File: scripts/release/publishpackages.mjs (hunk 1 above)
Comment (pomek) [reply scripts/release/publishpackages.mjs:89]: Review https://github.com/ckeditor/ckeditor5-dev/pull/1019, please.

**Review by martnpaneq — APPROVED** [review]

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Review by LukaszGudel — APPROVED** [review]

**Pull request merged**